File: 1013.py
#1013 O MAIOR
import math
valores = input().split(" ")

a = int(valores[0])
b = int(valores[1])
c = int(valores[2])

maiorAB = (a + b + (abs(a - b)))//2
maiorC = (c + maiorAB + (abs(c-maiorAB)))//2
print(maiorC, "eh o maior")

File: javascript/1013.js
/*O maior*/
const input = require("fs").readFileSync("/dev/stdin", "utf8");

const lines = input.split(" ");

const [A, B, C] = input.split(" ").map(item => parseInt(item));

const MaiorAB = (A + B + Math.abs(A - B)) / 2;
const MaiorABC = (C + MaiorAB + Math.abs(C - MaiorAB)) / 2;

console.log(MaiorABC + " eh o maior");